### ISSUE TITLE
Feature Request: Support multible payment methods

### ISSUE THREAD

**Issue by mojoaxel** [licensezero/licensezero-cli#6]
According to [the REDME](https://github.com/licensezero/licensezero-cli/blob/master/README.md#for-maintainers) you'll need a stripe account to receive payments. Also users will need a credit card.

Please add more payment-methods like maybe PayPal and most of all Cryptocoins like Litecoin, Dash and so on.

**Comment by kemitchell** [comment]
@mojoaxel, thanks for your message.

I'm open to supporting more payment methods through License Zero. For example, I would be very interested to find and integrate an API-accessible service to receive check or wire transfer payments on developers' behalf, so company customers can pay as they're currently used to paying outside vendors on invoices and purchase orders. That would help reduce friction for a significant class of potential customers, which would offset the time and complexity cost of supporting more than one payment method.

In the medium term, licensezero.com may support other payment methods through Stripe, especially ACH, especially for relicensing sponsorship payments, to make processing-fee savings available to developers. In the short term, however, I think credit card, via Stripe, will be the only supported method of payment. 

Stripe has some support for cryptocurrency, and may expand to cover other coins. But to be transparent, so you and others can make good decisions: I'm unlikely to support cryptocurrency payments via licensezero.com in the foreseeable future. A few reasons.

1. First and foremost, the purpose of License Zero is to demonstrate a model of long-term sustainability for open software development and a shift in how maintainers and consumers of open software relate, financially. If it succeeds, I wouldn't want its success clouded by close connection to cryptocurrencies, the speculative fervor around them, or the relative dearth of ways to spend them.

2. Cryptocurrency support, even through an API like Stripe, brings with it all the community, political, and technical complexity of each cryptocurrency supported. Those problems aren't related to License Zero's purpose. They would become distractions.

3. I keep up-to-date on the regulatory picture for cryptocurrencies and blockchain tokens, for other work that I do. I wouldn't want to complicate License Zero, either as the one operating it, or for those using it, with all of that uncertainty and potential compliance burden. As it is, License Zero leaves it up to licensors to decide and do what needs doing for tax compliance. In the future, License Zero may be able to automate or otherwise assist there, at least for some jurisdictions. In many jurisdictions, how to handle even simple credit card sales isn't clear. There are even more questions about how to handle cryptocurrency payments. And that's just tax.

4. I suspect the vast majority of potential customers who might pay with cryptocurrency also have credit cards.

In short, while I think cryptocurrencies and tokens are very interesting, License Zero is plenty interesting enough with a "boring" choice of payment method. If License Zero succeeds in getting developers supported with credit card payments, and establishes itself as a platform for the new sustainability model it proposes, in time, it may be worthwhile to support other payment methods, to optimize the low-friction payment experience. But right now, that kind of optimization is very premature.
